pkg/sdkserver/localsdk.go

```diff
@@ -384,28 +387,139 @@ func (l *LocalSDKServer) stopReserveTimer() {
+	defer l.gsMutex.Unlock()
+
+	if l.gs.Status.Players == nil {
+		l.gs.Status.Players = &sdk.GameServer_Status_PlayerStatus{}
+	}
+
+	found := -1
+	for i, playerID := range l.gs.Status.Players.IDs {
+		if playerID == id.PlayerID {
+			found = i
+			break
+		}
+	}
+	if found == -1 {
+		return &alpha.Bool{Bool: false}, nil
+	}
+
+	l.gs.Status.Players.IDs = append(l.gs.Status.Players.IDs[:found], l.gs.Status.Players.IDs[found+1:]...)
+	l.gs.Status.Players.Count = int64(len(l.gs.Status.Players.IDs))
+
+	l.update <- struct{}{}
+	l.recordRequestWithValue("playerdisconnect", "", "PlayerIDs")
+	return &alpha.Bool{Bool: true}, nil
+}
+
+// IsPlayerConnected returns if the playerID is currently connected to the GameServer.
+// [Stage:Alpha]
+// [FeatureFlag:PlayerTesting]
+func (l *LocalSDKServer) IsPlayerConnected(c context.Context, id *alpha.PlayerID) (*alpha.Bool, error) {
+	if !runtime.FeatureEnabled(runtime.FeaturePlayerTracking) {
+		return nil, errors.New(string(runtime.FeaturePlayerTracking) + " not enabled")
+	}
+
+	result := &alpha.Bool{Bool: false}
+	logrus.WithField("playerID", id.PlayerID).Info("Is a Player Connected?")
+	l.gsMutex.Lock()
+	defer l.gsMutex.Unlock()
+
+	l.recordRequestWithValue("isplayerconnected", id.PlayerID, "PlayerIDs")
+
+	if l.gs.Status.Players == nil {
+		return result, nil
+	}
+
+	for _, playerID := range l.gs.Status.Players.IDs {
+		if id.PlayerID == playerID {
+			result.Bool = true
+			break
+		}
+	}
+
+	return result, nil
 }
 
-// IsPlayerConnected returns if the player ID is connected or not
+// GetConnectedPlayers returns the list of the currently connected player ids.
 // [Stage:Alpha]
 // [FeatureFlag:PlayerTesting]
-func (l *LocalSDKServer) IsPlayerConnected(ctx context.Context, id *alpha.PlayerID) (*alpha.Bool, error) {
-	panic("implement me")
+func (l *LocalSDKServer) GetConnectedPlayers(c context.Context, empty *alpha.Empty) (*alpha.PlayerIDList, error) {
+	if !runtime.FeatureEnabled(runtime.FeaturePlayerTracking) {
+		return nil, errors.New(string(runtime.FeaturePlayerTracking) + " not enabled")
+	}
+	logrus.Info("Getting Connected Players")
+
+	result := &alpha.PlayerIDList{List: []string{}}
+
+	l.gsMutex.Lock()
+	defer l.gsMutex.Unlock()
+	l.recordRequest("getconnectedplayers")
+
+	if l.gs.Status.Players == nil {
+		return result, nil
+	}
+	result.List = l.gs.Status.Players.IDs
+	return result, nil
 }
 
-// GetConnectedPlayers returns if the players are connected or not
+// GetPlayerCount returns the current player count.
 // [Stage:Alpha]
 // [FeatureFlag:PlayerTesting]
-func (l *LocalSDKServer) GetConnectedPlayers(ctx context.Context, empty *alpha.Empty) (*alpha.PlayerIDList, error) {
-	panic("implement me")
+func (l *LocalSDKServer) GetPlayerCount(ctx context.Context, _ *alpha.Empty) (*alpha.Count, error) {
+	if !runtime.FeatureEnabled(runtime.FeaturePlayerTracking) {
+		return nil, errors.New(string(runtime.FeaturePlayerTracking) + " not enabled")
+	}
+	logrus.Info("Getting Player Count")
```

Comment:
I have added logger, so we can set log level.
```suggestion
	l.logger.Info("Getting Player Count")
```